### ISSUE TITLE
Unable to read schema with endpoints that require authorization

### ISSUE THREAD

**Issue by olvlvl** [graphcool/graphql-playground#164]
Hi,

Is there a way of providing an authorization token to Playground and be able to read schema on protected endpoints? For instance, I can query GitHub API v4 because I'm adding the `Authorization` field myself in the headers, but I can't provide that information to the application and the _schema_ section remains sadly empty.

I would like to have a field in the UI where I define the `Authorization` field that should be used to retrieve the schema, and send requests to the endpoint.

**Comment by kbrandwijk** [comment]
From what I picked up in another issue. If you specify an Authorization header on the query, then click 'reload schema', it will actually use that Authorization header for loading the schema.

**Comment by olvlvl** [comment]
I'm specifying the `Authorization` header to be able to query GitHub, but the schema panel remains empty.

**Comment by kbrandwijk** [comment]
No, it has nothing to do with Authorization. Github implemented their introspection query differently. Instead of POSTing the introspection query to the endpoint, and getting the results, the Github API expects a single `GET` request to the endpoint to get the introspection _results_, like described here: https://developer.github.com/v4/guides/intro-to-graphql/#discovering-the-graphql-api. This scenario is not covered by GraphQL-playground at the moment. 

@timsuchanek It would be interesting to see, if there is an easy way to allow 'custom' introspection queries, or if that's not possible, allow you to paste the introspection query results somewhere, so you can run the request outside of the playground.
